dotnet/WPF/VisualLayerIntegration/VisualLayerIntegration/BarGraphHostControl.xaml.cs

```diff
@@ -135,16 +134,18 @@ private SysWin.Point GetControlPointInDIP(UIElement control)
             // Top left of control relative to screen.
             var controlTopLeft = control.PointToScreen(new SysWin.Point(0, 0));
             // Convert screen coord to DIP.
```

Comment:
You conver the screen coordinates to DIP in at least 2 places. Create a method to do it rather than repeating the algorithm each time.

Reply:
updated